packages/permission-controller/src/PermissionController.test.ts

```diff
@@ -4334,6 +4370,67 @@ describe('PermissionController', () => {
         true,
       );
     });
+
+    it('correctly throws errors that do not inherit from JsonRpcError', async () => {
+      const options = getPermissionControllerOptions();
+      const { messenger } = options;
+      const origin = 'metamask.io';
+      const controller = getDefaultPermissionController(options);
+
+      const callActionSpy = jest
+        .spyOn(messenger, 'call')
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        .mockImplementationOnce(async (...args: any) => {
```

Comment:
why couldn't args be typed properly?

Reply:
I tried to add a more specific type, but got a pretty inscrutable error as a result. I could try to debug that, but if we somehow mess up these mocks, the tests ought to fail in unexpected ways, which we would immediately discover. It probably could be done, but I don't think it's worth our time.